CONTRIBUTING.md

```diff
@@ -1,18 +1,102 @@
+OpenStreetMap (and StreetComplete) works, because of a diverse community all over the world.
+
+[Especially look at the issues tagged with "help wanted"](https://github.com/westnordost/StreetComplete/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22).
+
+## Testing and reporting issues
+
+- [x] Requires no programming knowledge!
+- [x] Requires no OpenStreetMap knowledge!
+
+If you experience problems, crashes or a quest is not clear to you, feel free to open an issue for that. Remember to open one issue _for one matter_, so do not open one issue "I have found several problems", but one for each problem.
+
+1. If you have questions, remember to [read the FAQ](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ) first.
+2. Look [whether your issue has already been reported](https://github.com/westnordost/StreetComplete/issues).
+3. Now you can open a new issue. Give it a meaningful title and describe the problem. Do mention the StreetComplete and Android version, if it is a technical bug.
+
+### Map style issues
+
+Note that the map style used by StreetComplete is developed as a [separate project](https://github.com/ENT8R/streetcomplete-mapstyle), so please report issues related to it there.
+
+### Suggesting new quests
+
+We appreciate all quest ideas, you and others have, but some ideas cannot be added into the app, due to various design decicions and requirements. This is required in order to make a great app for all users of StreetComplete, whose contributions actually help the OpenStreetMap project. A quest, which nobody can solve and where users potentially choose wrong options and add wrong data to OpenStreetMap hurts not only OSM, but also this app.
+
+What you need to do:
+
+1. Search whether the quest you want to suggest is [already implemented in StreetComplete](https://wiki.openstreetmap.org/wiki/StreetComplete/Quests) or [already suggested in the issue tracker](https://github.com/westnordost/StreetComplete/issues).
+2. Please read the full guide on what requirements you need to consider to make the quest a great thing: [Adding new Quests to StreetComplete](https://github.com/westnordost/StreetComplete/wiki/Adding-new-Quests-to-StreetComplete).
+3. Open a new issue and fill out the issue template. Explain, why your quest meets the criteria you need to check, so other users can follow the example.
+4. Wait for input of others, even if you could implement it right away, it is always suggested to _first_ suggest a quest and implement it when it is accepted.
+
+➡️ If you can code, see also how to [develop your own quest](#developing-new-quests).
+
+## Improving documentation
+
+- [x] Requires no programming knowledge!
+- [ ] Requires no OpenStreetMap knowledge!
+
+You can also help to keep the wiki **up-to-date** for StreetComplete. For example:
```

Comment:
Perhaps clarify "the wiki" to "the OSM wiki" to differentiate it from the GitHub wiki?